packages/core/src/client.ts

```diff
@@ -290,69 +290,75 @@ export class SlashauthClient {
     );
   }
 
-  async addWalletToRoleForOrganization({
-    organizationID,
-    wallet,
+  async addAssignedRoleToUser({
```

Comment:
allows devs to assign wallet roles directly from the SDK for the app-level as well